types/util.promisify/index.d.ts

```diff
@@ -0,0 +1,39 @@
+// Type definitions for util.promisify 1.0
+// Project: https://github.com/ljharb/util.promisify#readme
+// Definitions by: Adam Voss <https://github.com/adamvoss>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/*~ Note that ES6 modules cannot directly export callable functions.
+ *~ This file should be imported using the CommonJS-style:
+ *~   import x = require('util.promisify');
+ *~
+ *~ Refer to the documentation to understand common
+ *~ workarounds for this limitation of ES6 modules.
+ */
+export = promifify;
+
+// tslint:disable-next-line ban-types
+declare function promifify(f: Function): Function;
```

Comment:
I'd like to see some justification for using `Function`. `() => void` or `(...args: any[]) => any` are usually better choices